readme.md

```diff
@@ -128,41 +128,67 @@ This is a one-time pay for each task. Payments will be in XMR.
-files to work with: src/qr.cpp; include/qr.hpp
-libraries used: libpng, QR Code generator
-difficulty: intermediate
-job status: not filled
-
-$250 - fix decoding error in functions using EVP_PKEY_encrypt() and EVP_PKEY_decrypt()
-files to work with: src/encryptor.cpp; include/encryptor.hpp
-libraries used: openssl
-difficulty: intermediate
-job status: not filled
-
-$250 - fetch a list of monero nodes
-libraries used: ?
-difficulty: intermediate
-job status: not filled
-
-$100 - create an organized documentation on the neroshop codebase including functions, classes, etc.
-tools to use: doxygen or any other tools
-difficulty: easy
-job status: not filled
+Details: Implement peer discovery system that broadcasts/multicasts messages to other peers, connects to other peers, stores their IP addresses and shares it with other neroshop nodes.
+Bounty Reward: $1000-$3000
+Files: src/daemon.cpp; src/server.cpp; src/client.cpp
+Libraries/Tools: System network libraries (Linux Sockets, Winsock2, etc.)
+Difficulty: Hard
+Status: Not Filled
+
+
+Details: Create an advance search engine using built-in LMDB database. More details will be discussed.
+Bounty Reward: $2000-$5000
+Files: src/database/lmdb.cpp; src/database/lmdb.hpp;
+Libraries/Tools: LMDB
+Difficulty: Extreme
+Status: Not Filled
+
+
+Details: Fix font loading issue where loading some fonts cause a crash
+Bounty Reward: $100
+Files: external/dokun-ui/src/font.cpp; external/dokun-ui/src/font.hpp
+Libraries/Tools: FreeType2
+Difficulty: Easy-Intermediate
+Status: Not Filled
+
+
+Details: Fix issue where Mouse presses affect all overlapping GUIs when it should ONLY affect the GUI on top
+Bounty Reward: $100
+Files: 
+Libraries/Tools: 
+Difficulty: Easy-Intermediate
+Status: Not Filled
+
+
+Details: Improve Input functions (Keyboard input is received/triggered multiple times even with a single key press on X11/Linux); Replace Mouse presses with complete clicks (Mouse press and release)
+Bounty Reward: $250
+Files: external/dokun-ui/src/mouse.cpp; external/dokun-ui/src/keyboard.cpp; external/dokun-ui/src/window.cpp
+Libraries/Tools: 
+Difficulty: Intermediate
+Status: Not Filled
+
+
+Details: Fix and optimize graphics and add graphical user interface features (such as scrollbar and scrollarea); Improve current GUI elements (specifically the text edit)
+Bounty Reward: $1000-$2000
+Files: external/dokun-ui/src/renderer.cpp; external/dokun-ui/include/renderer.hpp; 
+Libraries/Tools: OpenGL, Vulkan (optional)
+Difficulty: Intermediate-Hard
+Status: Not Filled
+
+
+Details: Fix decoding error in functions using EVP_PKEY_encrypt() and EVP_PKEY_decrypt()
+Bounty Reward: $100
+Files: src/encryptor.cpp; include/encryptor.hpp
+Libraries/Tools: OpenSSL
+Difficulty: Easy-Intermediate
+Status: Not Filled
+
+
+Details: Create an organized documentation on the neroshop codebase including functions, classes, etc.
+Bounty Reward: $50
+Libraries/Tools: Doxygen or any other tools
+Difficulty: Easy
+Status: Not Filled
```

Comment:
Can you take this off? I'd prefer doing it. Pretty easy and I want to do it in a way that produces a nice-looking doxygen output.

Reply:
Sure, I'll remove this specific bounty.